docs/src/pages/components/Switch.vue

```diff
@@ -36,6 +36,16 @@
           <md-switch v-model="checked7" id="my-test7" name="my-test7" class="md-primary" disabled>Light Blue Primary Color Disabled</md-switch>
         </div>
       </demo-example>
+
+      <demo-example label="Typed">
```

Comment:
The default behaviour will be ```type="button"``` and if the user change the type to ```submit``` in a switch inside a form this will submit the form. This will make things more explicit to a user that he needs to force the type to have a submit functionality, as the common scenario of a switch is to change a ```v-model``` value.